### ISSUE TITLE
Multiline javadoc comment crashes jdl generation

### ISSUE THREAD

**Issue by yelhouti** [jhipster/generator-jhipster#9331]
##### **Overview of the issue**

Multi line comments fails when javadoc contains new line
##### **Motivation for or Use Case**
Have well documented code

Should be working according to the doc
##### **Reproduce the error**
Generate from jdl with multiline comment:
```
     /**
     * Line1
     * Line2
     */
```

##### **JHipster Version(s)**

5.8.1, never seemed to work well for me
##### **Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**

File.json
```
{
    "name": "Customer",
    "fields": [
        {
            "fieldName": "name",
            "fieldType": "String",
            "fieldValidateRules": [
                "required"
            ]
        },
        {
            "fieldName": "businessId",
            "javadoc": "line1\nline2",
            "fieldType": "Integer",
            "fieldValidateRules": [
                "required"
            ]
        },
    ],
    "changelogDate": "20180429011313",
    "entityTableName": "customer",
    "dto": "mapstruct",
    "pagination": "no",
    "service": "serviceImpl",
    "jpaMetamodelFiltering": true,
    "fluentMethods": true,
    "clientRootFolder": "",
    "applications": "*",
    "microserviceName": "microservice1"
}
```
##### **Browsers and Operating System**

-   [X] Checking this box is mandatory (this is just to show you read everything)

**Comment by henkelb** [comment]
Reproduced this too since passed to jhispter 5.8.1.

**Comment by jdubois** [comment]
@MathieuAA isn't this something you should handle on JHipster Core?

**Comment by MathieuAA** [comment]
Yes indeed.Already on it, just forgot to assign myself on this issue

**Comment by jdubois** [comment]
Thanks a lot @MathieuAA !

**Comment by MathieuAA** [comment]
Can't reproduce this one. Can anyone post a **complete** JDL sample for me? I've tried with
```
/**
* AAA
* BBB
*/
```

with and without something after

**Comment by pmverma** [comment]
@MathieuAA I had this same problem. But only for field comments.

**Comment by MathieuAA** [comment]
Okay that may explain why I could reproduce it but thought I must have messed up somehow because I couldn't today... Let me try again

**Comment by MathieuAA** [comment]
Again, there's no error...
Here's what I'm gonna do:
  - write some heavy tests and see if, where and why it fails
  - if there's nothing, I'm gonna close this

I'm not excluding one commit must have fixed it without intent, between the last release and master I mean. So I'll do a release just to be sure.

**Comment by MathieuAA** [comment]
Closing this. Feel free to open a new issue on JHipster-Core, with a failing JDL sample if anyone finds anything. I've tried with entities, fields, relationships...

**Comment by pmverma** [comment]
I had problem with 5.8.1, I have not tried with master branch.

**Comment by yelhouti** [comment]
@MathieuAA you need to enable translation, the problem happens with prettier in the i18n .json files, more precisely, json files don't allow a line return in the middle of the value ex:
```
{
    "key": "multiline
value"
}
``` 
Instead we should have:

```
{
    "key": "multiline\\ nvalue"
}
```
you therefore only need to replace all `\n` by `\\n` in [field/relashionship].javadoc by writing the json files
